pkgs/applications/graphics/displaycal/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
```

Comment:
Take `python3` as parameter instead of all the Python packages/functions.

Reply:
Might we want the Python 3.10 specifically since the derivation is doing some linking to the versioned directory?